aten/src/ATen/native/cuda/PowKernel.cu

```diff
@@ -156,11 +156,18 @@ void pow_tensor_scalar_kernel_impl(TensorIterator& iter,
   }
 }
 
+#if defined(CUDA_VERSION) && (CUDA_VERSION < 10000)
+#define CUDA92_BUG(x) thrust::complex<T>(x.real(), x.imag())
+#else
+#define CUDA92_BUG(x) x
+#endif
+
 void pow_tensor_scalar_kernel(TensorIterator& iter, Scalar exp_scalar) {
   if (isComplexType(iter.dtype()) || exp_scalar.isComplex()) {
     AT_DISPATCH_COMPLEX_TYPES(iter.dtype(), "pow_cuda", [&]() {
       using thrust_t = typename ztype_cuda<scalar_t>::thrust_t;
```

Comment:
It is not easy to migrate this from `thrust::complex` to `c10::complex` due to bugs on `std::pow` on Windows nvcc, and I don't want this PR to be blocked by this. So in this PR, I choose to keep `thrust::complex` and workaround the `c10::complex --> thrust::complex` cast.